jsonnet/main.jsonnet

```diff
@@ -106,11 +106,16 @@ local inCluster =
         image: $.values.common.images.grafana,
         commonLabels+: $.values.common.commonLabels,
         prometheusName: $.values.common.prometheusName,
+
+        local etcdMixin = (import 'github.com/etcd-io/etcd/contrib/mixin/mixin.libsonnet') + {
```

Comment:
Why move it here? Wouldn't it be simpler to just do `etcdPrometheusRule:: null` in `jsonnet/control-plane.libsonnet`? This way we still have dashboard but there is no PrometheusRule objects which seems to be what we want.